lesson02/launch_confg.tf

```diff
@@ -1,9 +1,5 @@
 variable "instance_type" {
-  default = "t2.micro"
-}
-
-variable "key_name" {
-  default = "academy-ohio"
+  default = "m4.large"
```

Comment:
Since the participants will pay for their instances, should we update this to a cheaper instance? e.g. `t3.medium` or `t3.large`

Reply:
I think `t3.medium` could be a good fit. Anyways we need to remember everyone to do `terraform destroy`.

Reply:
t3 and t2 need special request from AWS it took me 2 days to get them approved, that's why I added a `m4.large` let's keep this one and use it as spot